packages/ui-a11y-utils/src/KeyboardFocusRegion.ts

```diff
@@ -41,55 +41,49 @@ import { logError as error } from '@instructure/console'
 import keycode from 'keycode'
 
 import { scopeTab } from './scopeTab'
+import type { UIElement } from '@instructure/shared-types'
+import { FocusRegionOptions } from './FocusRegionOptions'
 
 class KeyboardFocusRegion {
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'element' implicitly has an 'any' type.
-  constructor(element, options) {
-    this._contextElement = findDOMNode(element)
-    // @ts-expect-error ts-migrate(2339) FIXME: Property '_options' does not exist on type 'Keyboa... Remove this comment to see the full error message
+  private readonly _options: FocusRegionOptions
+  private _focusLaterElement: Element | null = null
+  private _needToFocus = false
+  private _listeners: ReturnType<typeof addEventListener>[] = []
+  private _raf: RequestAnimationFrameType[] = []
+  private _active = false
+  private _wasDocumentClick?: boolean
+  public _contextElement?: Node
+
+  constructor(element: UIElement, options: FocusRegionOptions) {
+    this._contextElement = findDOMNode(element) as Node
     this._options = options || {
       shouldContainFocus: true,
       shouldReturnFocus: true,
-      // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-      onBlur: (event) => {},
-      // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-      onDismiss: (event) => {},
       defaultFocusElement: null
     }
-
-    // @ts-expect-error ts-migrate(2339) FIXME: Property '_options' does not exist on type 'Keyboa... Remove this comment to see the full error message
     if (this._options.shouldReturnFocus) {
-      this._focusLaterElement = this.activeElement
+      this._focusLaterElement = getActiveElement(this.doc)
```

Comment:
Just curious: why did you remove the getter for this?

Reply:
It was not used anywhere

Reply:
It was used in `focusDefaultElement() {}` :) I prefer writing getters if something is used in 2+ places, but can live with it like this :)